operator/pkg/component/component.go

```diff
@@ -203,6 +215,17 @@ var AllComponents = []Component{
 		FlattenValues:        true,
 		ReleaseName:          "ztunnel",
 	},
+	{
+		UserFacingName:       ZtunnelWindowsComponentName,
+		SpecName:             "ztunnel-windows",
+		ResourceType:         "DaemonSet",
+		ResourceName:         "ztunnel-windows",
+		HelmSubdir:           "ztunnel-windows",
+		ToHelmValuesTreeRoot: "ztunnel-windows",
+		ContainerName:        "istio-proxy-windows",
```

Comment:
The whole pod will be windows so probably easier to keep the same container name as Linux so tooling doesn't have to change